yarn-project/aztec-nr/aztec/src/context/public_vm.nr

```diff
@@ -0,0 +1,54 @@
+use dep::protocol_types::address::{
+    AztecAddress,
+    EthAddress,
+};
+
+// Getters that will be converted by the transpiler into their
+// own opcodes
+struct PublicVmContext {}
+
+impl PublicVmContext  {
+    // Empty new function enables retaining context.<value> syntax
+    pub fn new() -> Self {
+        PublicVmContext {}
+    }
```

Comment:
I agree that "public vm" is clearer and yet we've been saying AVM everywhere.... :man_facepalming: should we change our minds and call it "public vm"? :laughing:

Reply:
Oh lol, before seeing this i switched everything back to AVM lmao